packages/website/src/types.ts

```diff
@@ -7,7 +7,7 @@ export interface Member {
   isLead?: boolean;
   roleId: string;
   otherSubteams?: string[] | string | undefined;
-  subteam: string;
+  subteam?: string;
```

Comment:
Should business be a subteam, just like `Leads`?

Reply:
potentially, I just updated the type because the current data breaks parts of the site.